app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<section class="categories--container">
+  <article class='form--header'>
+    <%= link_to categories_path do %>
+      <i class="fa-solid fa-arrow-left"></i>
+    <% end %>
+    <h2><%= "#{@category.name}'s" %> Transactions </h2>
+  </article>
+   <p class="categories--total--amount">
+    <span>Total Amount:</span> 
+    <span>$<%= format_amount(@category.expenses.sum(:amount)) %></span>
+   <p>
+  <article class="catergories--list--container">
+    <ul class="categories--list">
+      <% @expenses.each do |expense| %>
+          <li class="categories--item">
+            <div class="categories--item--info">
+              <% if @category.icon.attached? %>
+              <%= image_tag @category.icon %>
+              <% else %>
+                <%= image_tag "https://media.designrush.com/inspiration_images/134933/conversions/_1511456189_555_McDonald's-preview.jpg" %>
+              <% end %>
+              <p><%= expense.name %></p>
+            </div>
+            <p class="categories--item--price">$<%= format_amount(expense.amount) %></p>
+        </li>
+      <% end %>
+    </ul>
+  </article>
+
+  <article class="categories--btn--container">
+    <%= link_to 'Add a New Transaction', new_category_expense_path(@category), class: "add--btn" %>
+  </article>
```

Comment:
- [ ] As required for the project your app to follow the the [the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) from Behance, kindliy ensure that you update your current design by using the typography(font, size, weight,...), text-transformation, Layout(alignment and position of elements), colors, etc.

| Template design | your Design | 
| ---------------------- | -------------------- |
| ![Screenshot from 2023-11-21 17-22-34](https://github.com/Akorede3133/expense-insight/assets/83042742/fa478296-2a85-4de3-9f58-baa024a17dac) | ![image](https://github.com/Akorede3133/expense-insight/assets/83042742/e0fca99f-c725-43b6-abb8-55660915f22f) ||